KerbalStuff/blueprints/anonymous.py

```diff
@@ -29,7 +29,7 @@ def index():
     featured = Featured.query.order_by(desc(Featured.created)).limit(6)[:6]
     top = search_mods("", 1, 3)[0]
     new = Mod.query.filter(Mod.published).order_by(desc(Mod.created)).limit(3)[:3]
```

Comment:
Would it make sense to limit new mods to mods that are truly new, IE only have one version?